lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -148,8 +148,8 @@ def test_valid_user_delete(client, server_config, pytestconfig):
             result = TestUserManagement.register_user()
 
             result = runner.invoke(user_delete, args=[TestUserManagement.USER_TEXT])
-            assert result.exit_code == 0, result.stderr
-            assert result.stdout == "User test_user deleted\n"
+            assert result.exit_code == 0
```

Comment:
You should revert this change:  if the exit code is _not_ zero, then the reason for the error _should_ appear in `result.stderr`; and, if the exit code _is_ zero, the assertion won't fail.